Dockerfile.static-libs

```diff
@@ -0,0 +1,90 @@
+#                 _
+#                | |
+#  _ __ _   _ ___| |_
+# | '__| | | / __| __|
+# | |  | |_| \__ \ |_
+# |_|   \__,_|___/\__|
+
+FROM alpine:3.12 as rust-builder
+
+RUN apk update &&\
+    apk add --no-cache git gcc g++ make build-base openssl-dev musl musl-dev curl
+
+RUN curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y
+RUN /root/.cargo/bin/rustup target add $(uname -m)-unknown-linux-musl
+
+RUN wget https://github.com/libunwind/libunwind/releases/download/v1.3.1/libunwind-1.3.1.tar.gz && \
+    tar -zxf libunwind-1.3.1.tar.gz && \
+    cd libunwind-1.3.1/ && \
+    ./configure --disable-minidebuginfo --enable-ptrace --disable-tests --disable-documentation && make && make install
+
+COPY third_party/rustdeps /opt/rustdeps
+
+WORKDIR /opt/rustdeps
+
+RUN RUSTFLAGS="-C target-feature=+crt-static" /root/.cargo/bin/cargo build --release --target $(uname -m)-unknown-linux-musl
```

Comment:
If musl is used, we will not be able to use this library later in any dynamic library.